src/wast-parser.cc

```diff
@@ -3100,10 +3159,9 @@ Result WastParser::ParseBlockDeclaration(BlockDeclaration* decl) {
   WABT_TRACE(ParseBlockDeclaration);
   FuncDeclaration func_decl;
   CHECK_RESULT(ParseTypeUseOpt(&func_decl));
```

Comment:
Can we simply remove `func_decl` and just use `decl` for the `ParseTypeUseOpt` call too?

Reply:
It looks like the two structures are the same.